bundles/pixi.js-node/package.json

```
@@ -0,0 +1,80 @@
+{
+  "name": "@pixi/node",
+  "version": "6.4.2",
+  "description": "Bundle for PixiJS with support for NodeJS",
+  "homepage": "http://pixijs.com/",
+  "bugs": "https://github.com/pixijs/pixi.js/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pixijs/pixi.js.git"
+  },
+  "license": "MIT",
+  "author": "PixiJS Team",
+  "main": "dist/cjs/node.cjs",
+  "module": "dist/esm/node.mjs",
+  "types": "index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./index.d.ts",
+        "default": "./dist/esm/node.mjs"
+      },
+      "require": {
+        "types": "./index.d.ts",
+        "default": "./dist/cjs/node.cjs"
+      }
+    }
+  },
+  "files": [
+    "dist",
+    "*.d.ts"
+  ],
+  "engines": {
+    "node": ">=16"
+  },
+  "dependencies": {
+    "@types/gl": "^4.1.0",
+    "@types/xml2js": "^0.4.11",
+    "canvas": "^2.9.1",
+    "cross-fetch": "^3.1.5",
+    "gl": "^5.0.0",
+    "xml2js": "^0.4.23"
+  },
+  "peerDependencies": {
```

For bundles the packages should be dependencies, not peerDependencies.
```suggestion
    "xml2js": "^0.4.23",
```